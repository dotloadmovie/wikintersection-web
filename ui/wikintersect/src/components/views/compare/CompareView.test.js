import React from 'react';
import {CompareViewContainer} from './CompareView.container';

import { shallow } from 'enzyme';
import {SearchViewContainer} from "../search/SearchView.container";

it('displays properly with a result set', () => {
    const props = {};
    props.results = ["386BSD","A/UX","Android (operating system)","Apache HTTP Server","AudioOS","BSD/OS","BSD Daemon"];

    const component = shallow(<CompareViewContainer {...props}/>)

    expect(component).toMatchSnapshot();
})