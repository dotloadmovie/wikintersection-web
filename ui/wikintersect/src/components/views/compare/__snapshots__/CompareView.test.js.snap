// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`displays properly with a result set 1`] = `
<div>
  <Table
    bordered={false}
    className=""
    columns={
      Array [
        Object {
          "dataIndex": "count",
          "key": "count",
          "title": "#",
        },
        Object {
          "dataIndex": "value",
          "key": "value",
          "title": "Article",
        },
      ]
    }
    dataSource={
      Array [
        Object {
          "count": 1,
          "key": 0,
          "value": "386BSD",
        },
        Object {
          "count": 2,
          "key": 1,
          "value": "A/UX",
        },
        Object {
          "count": 3,
          "key": 2,
          "value": "Android (operating system)",
        },
        Object {
          "count": 4,
          "key": 3,
          "value": "Apache HTTP Server",
        },
        Object {
          "count": 5,
          "key": 4,
          "value": "AudioOS",
        },
        Object {
          "count": 6,
          "key": 5,
          "value": "BSD/OS",
        },
        Object {
          "count": 7,
          "key": 6,
          "value": "BSD Daemon",
        },
      ]
    }
    indentSize={20}
    loading={false}
    locale={Object {}}
    pagination={
      Object {
        "pageSize": 100,
      }
    }
    rowKey="key"
    showHeader={true}
    size="default"
    sortDirections={
      Array [
        "ascend",
        "descend",
      ]
    }
    useFixedHeader={false}
  />
</div>
`;
